extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -40,4 +43,129 @@ amp-story-subscriptions.i-amphtml-story-subscriptions-visible::before {
 amp-subscriptions-dialog.i-amphtml-story-subscriptions-dialog-wrapper {
   border-radius: 16px 16px 0px 0px !important;
   font-family: Poppins, sans-serif !important;
+  font-weight: 400 !important;
+  background: white !important;
+  color: black !important;
+}
+
+.i-amphtml-story-subscriptions-dialog-banner {
+  position: relative !important;
+  width: 100% !important;
+  border-radius: 16px 16px 0px 0px !important;
```

Comment:
This is brittle since we would need to update the border radius of the parent and this element if the border radius on the parent changes.

I would suggest putting `overflow: hidden` on the parent as long as the container doesn't need to scroll.

Reply:
Nice catch!